server/controllers/adminCtrl.js

```diff
@@ -263,11 +263,11 @@ async function getIndex(req, res, next) {
       /** ------- Got course code -> prepare for Page 1 depending on status (draft or published) ------- */
       log.debug(' getIndex, get course data for : ' + courseCode)
       const koppsApiResponse = await koppsCourseData.getKoppsCourseData(courseCode, lang)
-      if (koppsApiResponse.statusCode >= 400) {
+      if (koppsApiResponse.statusCode && koppsApiResponse.statusCode >= 400) {
```

Comment:
`statusCode` does not exist inside `body` it exists in `response of await`, it means it has to be destructured inside of `getKoppsCourseData` alongside with `body`, `const { body, statusCode } = await koppsApi.getAsync()` and return {body, statusCode}. Otherwise this piece of code will never have a statusCode

Reply:
done